doc/source/Architecture.rst

```diff
@@ -1,3 +1,24 @@
-------------
 Architecture
-------------
+============
+
+ the Rucio system is based on a distributed architecture and can be decomposed into four
+ main components: Server, Daemons, Resources, and User Interface.
+
+The Rucio server is a passive component listening to incoming queries. It connects several
+Rucio core components together and offers a common interface for external interaction.
+This abstraction layer is responsible for the interactions with different
```

Comment:
I think with "This Abstraction layer" RSEs are being ment? There is something missing here.